### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 48

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#236] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255572-f700c824-8969-43e5-8666-99f59fc847b8.png)

**Comment by i0nut02** [comment]
1. (20 scelgo 6)
2. (10 scelgo 3) * (10 scelgo 3) * (6 scelgo 1)
3. (10 scelgo 3) * (10 scelgo 3) * (3 scelgo 1) * (3 scelgo 1)

**Comment by lord-marcel** [comment]
La soluzione di Ionut è corretta. Il problema è analogo a quello della scelta dei rappresentanti della dispensa 3. Allego i due modi studiati per risolvere il problema.
![4444_page-0001](https://user-images.githubusercontent.com/36630876/152654207-79a80eb6-abf9-4f6c-891d-34468bda0a85.jpg)

**Comment by Exyss** [comment]
Soluzioni analoghe a @RiaMeLa
